src/layouts/Layout.vue

```diff
@@ -1,87 +1,157 @@
+    <header
+        v-else
+        class="d-flex flex-wrap justify-content-center mt-3 mb-3"
+    >
+        <router-link
+            to="/dashboard"
+            class="d-flex align-items-center  text-dark text-decoration-none"
+        >
+            <object
+                class="bi"
+                width="40"
+                height="40"
+                data="/icon.svg"
+            />
             <span class="fs-4 title ms-2">Uptime Kuma</span>
         </router-link>
     </header>
 
     <main>
         <!-- Add :key to disable vue router re-use the same component -->
-        <router-view v-if="$root.loggedIn" :key="$route.fullPath" />
+        <router-view
+            v-if="$root.loggedIn"
+            :key="$route.fullPath"
+        />
         <Login v-if="! $root.loggedIn && $root.allowLoginDialog" />
     </main>
 
     <footer>
         <div class="container-fluid">
             Uptime Kuma -
             Version: {{ $root.info.version }} -
-            <a href="https://github.com/louislam/uptime-kuma/releases" target="_blank" rel="noopener">Check Update On GitHub</a>
+            <a
+                href="https://github.com/louislam/uptime-kuma/releases"
+                target="_blank"
+                rel="noopener"
+            >Check Update On GitHub</a>
         </div>
     </footer>
 
     <!-- Mobile Only -->
-    <div style="width: 100%;height: 60px;" v-if="$root.isMobile"></div>
-    <nav class="bottom-nav" v-if="$root.isMobile">
-        <router-link to="/dashboard" class="nav-link" @click="$root.cancelActiveList"><div>📊</div>Dashboard</router-link>
-        <a href="#" :class=" { 'router-link-exact-active' : $root.showListMobile } " @click="$root.showListMobile = ! $root.showListMobile"><div>📃</div>List</a>
-        <router-link to="/add" class="nav-link" @click="$root.cancelActiveList"><div>➕</div>Add</router-link>
-        <router-link to="/settings" class="nav-link" @click="$root.cancelActiveList"><div>🔧</div>Settings</router-link>
+    <div
+        v-if="$root.isMobile"
+        style="width: 100%;height: 60px;"
+    />
+    <nav
+        v-if="$root.isMobile"
+        class="bottom-nav"
+    >
+        <router-link
+            to="/dashboard"
+            class="nav-link"
+            @click="$root.cancelActiveList"
+        >
+            <div>📊</div>Dashboard
+        </router-link>
+        <a
+            href="#"
+            :class=" { 'router-link-exact-active' : $root.showListMobile } "
+            @click="$root.showListMobile = ! $root.showListMobile"
+        ><div>📃</div>List</a>
+        <router-link
+            to="/add"
+            class="nav-link"
+            @click="$root.cancelActiveList"
+        >
+            <div>➕</div>Add
+        </router-link>
+        <router-link
+            to="/settings"
+            class="nav-link"
+            @click="$root.cancelActiveList"
+        >
+            <div>🔧</div>Settings
+        </router-link>
```

Comment:
Multiple lines of attribute is not good in this case.

Reply:
While it does look a bit weird in this case, I do find it helpful when we inevitably add a bunch of classes and make the line super long. I don't know if there's an intermediate setting, perhaps allow more chars per line?

Reply:
This is the recommended settings from `plugin:vue/vue3-recommended`

Reply:
https://eslint.vuejs.org/rules/max-attributes-per-line.html

I can override default settings. Just give me numbers for https://eslint.vuejs.org/rules/max-attributes-per-line.html#options

Reply:
Prefer to off, it should be determined by human.

"vue/max-attributes-per-line": "off"

Reply:
>While it does look a bit weird in this case, I do find it helpful when we inevitably add a bunch of classes and make the line super long.

And git diff will be more readable

Reply:
> Prefer to off, it should be determined by human.
> 
> "vue/max-attributes-per-line": "off"

OK